integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CommonLoadUtils.scala

```diff
@@ -212,7 +212,7 @@ object CommonLoadUtils {
       options: Map[String, String]): util.Map[String, String] = {
     val tableProperties = table.getTableInfo.getFactTable.getTableProperties
     val optionsFinal = LoadOption.fillOptionWithDefaultValue(options.asJava)
-    //    EnvHelper.setDefaultHeader(sparkSession, optionsFinal)
+    EnvHelper.setDefaultHeader(SparkSQLUtil.getSparkSession, optionsFinal)
```

Comment:
please add a comment here that it is for legacy store. 

All data load in new store success without this change. so I commented this. If there was a comment why this is required. I would have kept it.

Reply:
done